src/layout/Sidebar.tsx

```diff
@@ -128,20 +129,23 @@ export const Sidebar = (): JSX.Element => {
         <div className="balance-wrapper flex-item">
           {walletState.walletStatus === 'LOADED' && (
             <>
-              <BalanceDisplay availableBalance={walletState.getOverviewProps().availableBalance} />
+              <BalanceDisplay availableBalance={walletState.availableBalance} />
+              {/* Faucet button */}
               <div className="main-buttons">
-                {networkName === 'testnet-internal-nomad' && (
-                  <Button
-                    data-testid="faucet-button"
-                    type="primary"
-                    className="faucet-button action left-diagonal"
-                    {...fillActionHandlers((): void => {
-                      shell.openExternal(FAUCET_URL)
-                    })}
-                  >
-                    {t(['wallet', 'button', 'getTestETC'])}
-                  </Button>
-                )}
+                {networkName === 'testnet-internal-nomad' &&
+                  isSome(walletState.availableBalance) &&
+                  walletState.availableBalance.value.isZero() && (
```

Comment:
Why is this condition introduced here? 🤔  It might be quite impossible to reach exactly 0 balance after you already have some tx. If we want condition like this here (I get that showing that button all the time is ugly :D), I would go for `availableBalance < 1`, since you get 1 ETC from faucet.

Reply:
My thinking was that the user doesn't need it after one use / if the user already has funds. I'll change this.